Alignment/OfflineValidation/test/GCP_defaultRange.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
does this file belong here?

Reply:
Indeed, this can be moved away. Please approve this PR: https://github.com/cms-data/Alignment-OfflineValidation/pull/3

Reply:
this PR can be tested with the cms-data one, but I would suggest to push also the txt files in `Alignment/OfflineValidation/data` in the same PR

Reply:
Yes it is tested, GCP unit test still works plus I moved file to local data dir.